src/redux/actions/user/index.js

```diff
@@ -11,46 +11,64 @@ import {
 } from '../types';
 import instance from '../../../config/axiosConfig';
 import { filterUserPayload } from '../../reducers/util';
+import { logout } from '../auth/authActionCreators';
 
-export const fetchUser = username => async (dispatch) => {
+const requestDeniedMessage = 'Request failed with status code 401';
```

Comment:
Have you checked out any naming convention guidelines for constants?

Reply:
It has been resolved